queries/glimmer/highlights.scm

```diff
@@ -1,113 +1,84 @@
 ; === Tag Names ===
+
 ; Tags that start with a lower case letter are HTML tags
 ; We'll also use this highlighting for named blocks (which start with `:`)
 ((tag_name) @tag
-  (#lua-match? @tag "^:?[%l]"))
-
+  (#match? @tag "^(:)?[a-z]"))
 ; Tags that start with a capital letter are Glimmer components
 ((tag_name) @constructor
-  (#lua-match? @constructor "^%u"))
+  (#match? @constructor "^[A-Z]"))
```

Comment:
No. That uses the vim regex engine, which kills performance.

Reply:
oh excellent! I wondered why this change existed -- I'll revert upstream, too